application/models/Question.php

```diff
@@ -214,6 +215,8 @@ public function rules()
                 'message' => sprintf(gT("Code: '%s' is a reserved word."), $this->title), // Usage of {attribute} need attributeLabels, {value} never exist in message
                 'except' => 'archiveimport'
             );
+            /* Don't save empty or 'core' question theme name */
```

Comment:
Why "core" is not a valid one? What is used instead?

Reply:
empty / null (default)

Maybe related ? https://bugs.limesurvey.org/view.php?id=15982

Reply:
> empty / null (default)
> 
> Maybe related ? https://bugs.limesurvey.org/view.php?id=15982

Hmm not sure...